test_data/mysqldump.test.out

```diff
@@ -41,6 +43,16 @@ INSERT INTO `cart` VALUES ('901e-a6cfc2b502dc','abc-123',1,'2020-07-20 05:10:26'
 /*!40000 ALTER TABLE `cart` ENABLE KEYS */;
 UNLOCK TABLES;
 
+DROP TABLE IF EXISTS `CART`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!40101 SET character_set_client = utf8 */;
+CREATE TABLE `CART` (
```

Comment:
I think it's worth adding a comment saying that this Table and IDX index are purposefully added with the same name differing only in cases.